google/cloud/storage/_signing.py

```diff
@@ -77,7 +77,7 @@ def get_signed_query_params_v2(credentials, expiration, string_to_sign):
               signed payload.
     """
     ensure_signed_credentials(credentials)
-    signature_bytes = credentials.sign_bytes(string_to_sign)
+    signature_bytes = credentials.sign_bytes(string_to_sign.encode("ascii"))
```

Comment:
See https://github.com/googleapis/python-storage/blob/master/google/cloud/storage/_signing.py#L628 for what we do in V4